app/views/budgets/edit.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1><%= t(".title") %></h1>
+<%= simple_form_for budget do |f| %>
+  <%= render "form", f: f %>
+<% end %>
```

Comment:
Again, I will pull the whole form into the `_form` partial

Reply:
Done sha: a3fcca6